apispec/apispec-model/src/main/scala/sttp/tapir/apispec/model.scala

```diff
@@ -11,6 +13,13 @@ sealed trait ExampleValue
 case class ExampleSingleValue(value: String) extends ExampleValue
 case class ExampleMultipleValue(values: List[String]) extends ExampleValue
 
-case class Tag(name: String, description: Option[String] = None, externalDocs: Option[ExternalDocumentation] = None)
+case class Tag(
+  name: String,
+  description: Option[String] = None,
+  externalDocs: Option[ExternalDocumentation] = None,
+  extensions: ListMap[String, ExtensionValue] = ListMap.empty
+)
 
 case class ExternalDocumentation(url: String, description: Option[String] = None)
+
+final case class ExtensionValue(value: String)
```

Comment:
any reason why this is `final` and the others are not? I'd keep consistency here :)

Reply:
No reason, adjusted